matplotlib2tikz/path.py

```diff
@@ -129,30 +133,39 @@ def draw_pathcollection(data, obj):
         else:
             draw_options.append('colormap/' + mycolormap)
 
-    if len(obj.get_sizes()) == len(dd):
-        # See Pgfplots manual, chapter 4.25.
-        # In Pgfplots, \mark size specifies raddi, in matplotlib circle areas.
-        radii = numpy.sqrt(obj.get_sizes() / numpy.pi)
-        dd = numpy.column_stack([dd, radii])
-        labels.append('sizedata' + 14*' ')
-        draw_options.extend([
-            'visualization depends on='
-            '{\\thisrow{sizedata} \\as\\perpointmarksize}',
-            'scatter/@pre marker code/.append style='
-            '{/tikz/mark size=\\perpointmarksize}',
-            ])
-
-    do = ' [{}]'.format(', '.join(draw_options)) if draw_options else ''
-    content.append('\\addplot{}\n'.format(do))
-
-    to = ' [{}]'.format(', '.join(table_options)) if table_options else ''
-    content.append('table{}{{%\n'.format(to))
-
-    content.append((' '.join(labels)).strip() + '\n')
-    fmt = (' '.join(dd.shape[1] * ['%+.15e'])) + '\n'
-    for d in dd:
-        content.append(fmt % tuple(d))
-    content.append('};\n')
+    if isContour:
+        paths = len(obj.get_paths())
+    else:
+        paths = 1
+
+    for i in range(0, paths):
+        if isContour:
+            dd = obj.get_paths()[i].vertices
```

Comment:
What is `dd` if `not is_contour`?

Reply:
Thank you for review. dd is set in line 93. In case of contourf, it's value is set to {0, 0}. dd could be redefined in line 100, however this never gets executed in contourf since obj.get_array() returns None. Therefore in line 143 we are just redefining the value of dd because is not set correctly above.